virt/getting_started/virt-getting-started.adoc

```diff
@@ -49,10 +49,7 @@ Create a virtual machine (VM):
 * xref:../../virt/virtual_machines/creating_vms_rh/virt-creating-vms-from-rh-images-overview.adoc#virt-creating-vms-from-rh-images-overview[Create a VM from a Red Hat image].
 ifndef::openshift-rosa,openshift-dedicated[]
 +
-You can create a VM by using a Red Hat template or an instance type.
-+
-:FeatureName: Creating a VM from an instance type
-include::snippets/technology-preview.adoc[]
+You can create a VM by using a Red Hat template or an xref:../../virt/virtual_machines/creating_vms_rh/virt-creating-vms-from-instance-types.adoc#virt-creating-vms-from-instance-types[instance type].
```

Comment:
Should this be a separate bullet, as it links to a different assembly than Create a VM from a Red Hat image? It is a touch confusing to me.

Reply:
Good spot. I think there's some rework that needs to be done here in general, because there is overlap between the procedures in the "custom images" and "RH images" directories, and I think the separation is doing more harm than good. Sadly. 

(I'm pretty sure that this link was attached to the previous bullet because the instance type assembly is in the "creating_vms_rh" directory.)

Reply:
I think I'm going to leave this as-is and try to address it in a follow-up PR.